spacemesh/v1/mesh_types.proto

```diff
@@ -99,3 +99,11 @@ message LayerStreamRequest {}
 message LayerStreamResponse {
     Layer layer = 1;
 }
+
+message EpochStreamRequest {
```

Comment:
i thought that we already have some api to query atxs, we could use that until there is time to design new.

i wanted to have something similar to TransactionResult for every object.

```go
type AtxService interface {
     Stream(Request) 
}

type Request struct {
    EpochStart, EpochEnd uint32 // includes all in range[EpochStart, EpochEnd)
    Smesher []byte // if not empty filters by smesher
}
```

and basically similar api for every other object that can be consumed for various purposes

Reply:
so if you look at api/node.go in go-spacemesh, there is no interface that allows the API to query all atx ids in an epoch.
what's currently available is for the caller to query full atx data by atx ids.

my assumption is also that we will rework something new for the api. current state is difficult for me to use. so this would be short-lived and replaced by v2.

Reply:
i'd really like the primitive types to be consumed directly in v2. (u32, u64, []byte)

Reply:
sounds fine to me